add news post for rocur relaunch (#374)
rladies/rladies.github.io: content/news/2025-01-22_rocur-relaunch/index.en.md

---
title: "We're Launching the R-Ladies Rotating Curation (RoCur) Account on Bluesky!"
author: "RoCur team"
type: blog
date: "2025-01-22"
image: img/bluesky_media_kit_logo_1.png
tags: 
- rocur
categories: 
- R-Ladies
summary: We're looking for curators for the We Are R-Ladies Bluesky account!
slug: rocur-relaunch-2025
---

![Bluesky logo. A white butterfly on a blue gradient.](img/bluesky_media_kit_logo_1.png)

**Caption:** Bluesky logo, sourced from the official [Bluesky media kit](https://drive.google.com/drive/folders/16mlJPfWNnc6jj-3vGZ88SFysIY13GBp0).

> We're absolutely thrilled to announce that the [R-Ladies Rotating Curation (RoCur) account](https://bsky.app/profile/weare.rladies.org) is now live on Bluesky! Each week, a new member of the R-Ladies community will take over the account to talk about all things R. This will be a fantastic opportunity for individuals who identify as a minority gender in the R community to amplify their voices and connect with others in the greater data science community.

Reviewer: ```suggestion
We are absolutely thrilled to announce the R-Ladies Rotating Curation (RoCur) Bluesky account [bsky.app/profile/weare.rladies.org](https://bsky.app/profile/weare.rladies.org) is now live! Each week, a new member of the R-Ladies community will take over the account to talk about all things R. This will be a fantastic opportunity for individuals who identify as a minority gender in the R community to amplify their voices and connect with others in the greater data science community.
```

## History

Some of you might recall the RoCur account on Twitter. The program ran for about five years, giving over 100 curators the chance to take over the account and share their stories. While the Twitter account is no longer active, the archive of posts can still be viewed [here](https://rladies.org/tweet-archive/wearerladies/).

## How You Can Help

> We're excited to revive the RoCur program on Bluesky, picking up where we left off. You can help us build back the community stronger than ever by signing up to be a curator or nominating someone to be a curator!

Reviewer: ```suggestion
We are excited to revive the RoCur program on Bluesky, picking up where we left off. You can help us build back the community stronger than ever by signing up to be a curator or nominating someone to be a curator!
```

## Become a Curator

> *Are you interested in becoming a curator?* Curating is a great way to build your social media presence, develop your science communication skills, expand your CV, and connect with a supportive community. Sign up to be a curator on [this form](https://airtable.com/appjkZZgtF0iEWFjx/pagqsAma1WmUYxa9j/form). We're currently looking for curators until the middle of 2025, and we'll open up more spots as the year progresses.

Reviewer: ```suggestion
*Are you interested in becoming a curator?* Curating is a great way to build your social media presence, develop your science communication skills, expand your CV, and connect with a supportive community. Sign up to be a curator on the [@weare.rladies.org Curator Sign Up form](https://airtable.com/appjkZZgtF0iEWFjx/pagqsAma1WmUYxa9j/form). We are currently looking for curators until the middle of 2025, and we will open up more spots as the year progresses.
```

Reviewer: also, I'm not sure about the "expand your CV" part...

you could also add something about learning and sharing

> *Are you interested in becoming a curator?* Curating is a great way to build your social media presence, develop your science communication skills, expand your CV, and connect with a supportive community. Sign up to be a curator on [this form](https://airtable.com/appjkZZgtF0iEWFjx/pagqsAma1WmUYxa9j/form). We're currently looking for curators until the middle of 2025, and we'll open up more spots as the year progresses.

Reviewer: What about something like "We welcome curators of all experience levels, from those new to R to those with years of R experience. We also welcome curators of any native language - curation is not restricted to English."

## Nominate Someone

> *Do you know someone who'd be a perfect curator?* Perhaps a colleague, someone from your local R-Ladies chapter, or a user you can't get enough of on Bluesky! You can help us identify fantastic curators by nominating them. Submit your nominations on [this form](https://airtable.com/appFIBzsk2AxWlGMt/pagfezjuCxohikOOa/form). You can nominate as many people as you'd like, so don't hesitate to share this opportunity.

Reviewer: ```suggestion
*Do you know someone who would be a great curator?* Perhaps a colleague, someone from your local R-Ladies chapter, or a user you can't get enough of on Bluesky! You can help us identify fantastic curators by nominating them. Submit your nominations on the [@weare.rladies.org Curator Nomination Form](https://airtable.com/appFIBzsk2AxWlGMt/pagfezjuCxohikOOa/form). You can nominate as many people as you would like, so do not hesitate to share this opportunity.
```

> If you have any questions, feel free to reach out to the RoCur admins at [weare\@rladies.org](mailto:weare@rladies.org) or shoot a message on the **#rocur** channel on the [R-Ladies Community Slack](https://rladies.org/form/community-slack/).

Reviewer: ```suggestion
If you have any questions, feel free to reach out to the RoCur admins at [weare\@rladies.org](mailto:weare@rladies.org) or send a message on the **#rocur** channel on the [R-Ladies Community Slack](https://rladies.org/form/community-slack/).
```

Reviewer: oh and in this section maybe also link to the guide for more information

---

> title: "We're Launching the R-Ladies Rotating Curation (RoCur) Account on Bluesky!"

Reviewer 2: ```suggestion
title: "We're Launching the R-Ladies Rotating Curation Account on Bluesky (RoCur)!"
```

I feel that where it is right now, "(RoCur)" breaks the flow.

Author: Removing abbreviation all together from the title; "RoCur" will be introduced in the first sentence of the main body instead

author: "RoCur team"
type: blog

> date: "2025-01-22"

Reviewer 2: needs to be changed.

Author: Author changed; does anything else need to be changed here?

Reviewer: i updated the publishing date to 1/24 (i pushed the commit)

image: img/rladies-bluesky.jpg
tags: 
- rocur
categories: 
- R-Ladies
summary: We're looking for curators for the We Are R-Ladies Bluesky account!
slug: rocur-relaunch-2025
---

![Screenshot of We Are R-Ladies on Bluesky](img/rladies-bluesky.jpg)

> We are absolutely thrilled to announce the R-Ladies Rotating Curation (RoCur) Bluesky account [bsky.app/profile/weare.rladies.org](https://bsky.app/profile/weare.rladies.org) is now live! Each week, a new member of the R-Ladies community will take over the account to talk about all things R. This will be a fantastic opportunity for individuals who identify as a minority gender in the R community to amplify their voices and connect with others in the greater data science community.

Reviewer 2: For future PRs I'd recommend starting a new line after each sentence. 
You can use https://github.com/ropenscilabs/aeolus to fix documents :smile_cat:

## History

> Some of you might recall the RoCur account on Twitter. The program ran for about five years, giving over 100 curators the chance to take over the account and share their stories. While the Twitter account is no longer active, the archive of posts can still be viewed [here](https://rladies.org/tweet-archive/wearerladies/).

Reviewer 2: ```suggestion
Some of you might recall the RoCur account on Twitter. The program ran for about five years, giving over 100 curators the chance to take over the account and share their stories. While the Twitter account is no longer active, the archive of posts can still be [viewed](https://rladies.org/tweet-archive/wearerladies/).
```

https://www.boia.org/blog/click-here-isnt-enough-context-for-hyperlinks

## How You Can Help

We are excited to revive the RoCur program on Bluesky, picking up where we left off. You can help us build back the community stronger than ever by signing up to be a curator or nominating someone to be a curator!

## Become a Curator

> *Are you interested in becoming a curator?* Curating is a great way to share and learn new R techniques, build your social media presence, develop your science communication skills, and connect with a supportive community. We welcome curators of all experiences levels and professional backgrounds. Curation may be conducted in any language; posts are not restricted to English!

Reviewer 2: Just a note, if someone starts curating in say French, will the default language on posts be changed? To make it less likely they post without specifying it is French? (if the post is correctly classified in a language, then people can click on "translate")

Author: Thanks for the tip! If a potential curator reaches out about posting in a language other than English, we'll make sure to update the [FAQ page](https://guide.rladies.org/rocur/faq/) to show how you can specify the language(s) that's used in the post.
